lib/cdo/workshop_constants.rb

```diff
@@ -0,0 +1,35 @@
+module WorkshopConstants
```

Comment:
No idea. These were in ActivityConstants before, which was not the right place. It follows that shared is not the right place either. I'll move them to dashboard.